### DISCUSSION TITLE
can you help me about installing error?

### DISCUSSION THREAD

**Discussion by bunnywonyo** [Homebrew/discussions#3332] [Everyday usage]
bunny@MacBook-Pro ~ % brew install ffmpeg
Warning: No available formula with the name "ffmpeg".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
bunny@MacBook-Pro ~ % brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

I am trying to download ffmpeg and YouTube-dl and yt-dlp, I could download it yesterday, but not today. 
what can I do?

**Comment by SMillerDev** [comment]
Have you tried following the instructions in the last few lines of your post?
